src/UIAutomation/Microsoft.UI.UIAutomation/Microsoft.UI.UIAutomation.idl

```diff
@@ -560,6 +574,52 @@ namespace Microsoft.UI.UIAutomation
         AutomationRemoteBool IsNotEqual(AutomationRemoteOutlineStyles rhs);
     };
 
+    enum AutomationPatternId
+    {
+        Invoke = 10000,
+        Selection = 10001,
+        Value = 10002,
+        RangeValue = 10003,
+        Scroll = 10004,
+        ExpandCollapse = 10005,
+        Grid = 10006,
+        GridItem = 10007,
+        MultipleView = 10008,
+        Window = 10009,
+        SelectionItem = 10010,
+        Dock = 10011,
+        Table = 10012,
+        TableItem = 10013,
+        Text = 10014,
+        Toggle = 10015,
+        Transform = 10016,
+        ScrollItem = 10017,
+        LegacyIAccessible = 10018,
+        ItemContainer = 10019,
+        VirtualizedItem = 10020,
+        SynchronizedInput = 10021,
+        ObjectModel = 10022,
+        Annotation = 10023,
+        UIA_TextPattern2Id = 10024,
```

Comment:
>UIA_TextPattern2Id [](start = 8, length = 18)

There are a few enum variants that different than the others :)

(Seems like maybe the ones that have a number in the name?)

Reply:
It must be the fact that the name doesn't end with `PatternId`. I'm pretty sure I did a special case for this somewhere else in the code generator.

Reply:
Lol so there are. Thanks for checking this. I'll fix it up.